R/analyzer.R

```diff
@@ -40,4 +42,60 @@ get_energia <- function(data) {
     nrow()
   
   eventos / nrow(data)
+}
+
+#' @title Extrai o regime de tramitação de um PL
+#' @description Obtém o regime de tramitação de um PL
+#' @param tram_df Dataframe da tramitação do PL.
+#' @return String com a situação do regime de tramitação da pl.
+#' @examples
+#' extract_regime_senado(fetch_tramitacao(91341,'senado'))
```

Comment:
Não roda esse exemplo ta errado o nome da função e depois que eu rodei o extract_regime_tramitacao(fetch_tramitacao(91341,'senado')) aparace o erro: Error: Column `casa` not found
Mesma coisa para o extract_forma_apreciacao

Reply:
Documentação corrigida.